appserver/admingui/common/src/main/resources/security/realms/realmjs.inc

```diff
@@ -118,8 +118,12 @@ function showDisplay(testtype) {
     document.getElementById('#{certSectionId}').style.display = 'none'; 
     document.getElementById('#{solarisSectionId}').style.display = 'none';
     document.getElementById('#{pamSectionId}').style.display = 'none';
-    
-    if (testtype.match('none')){  
+    var loginModuleJaax = document.getElementById('#{loginModuleJaax}');
+
+    loginModuleJaax.style.display = 'none';
```

Comment:
For predefined classes, that already have an attribute for JAAS Context, it is not displayed for login module configuration:

![image](https://user-images.githubusercontent.com/1588543/54518783-70880e00-4965-11e9-842e-932fcc0bb1c2.png)